padd.sh

```diff
@@ -1413,8 +1413,12 @@ truncateString() {
 # Converts seconds to days, hours, minutes
 # https://unix.stackexchange.com/a/338844
 convertUptime() {
-    # shellcheck disable=SC2016
-    eval "echo $(date -ud "@$1" +'$((%s/3600/24)) days, %H hours, %M minutes')"
+
+  local D=$(($1/60/60/24))
```

Comment:
Shouldn't indentation use 4 spaces?

Reply:
Thanks. I took the reminder to fix the indention throughout the whole code.